cmd/apps.go

```diff
@@ -362,21 +362,27 @@ func updateApp(cmd *cobra.Command, args []string, appType string) error {
 	}
 
 	c := newClient(flagDomain, appType)
-	manifest, err := c.UpdateApp(&client.AppOptions{
+	opts := &client.AppOptions{
 		AppType:   appType,
 		Slug:      args[0],
 		SourceURL: src,
 
 		OverridenParameters: overridenParameters,
-	}, flagSafeUpdate)
+	}
+	manifest, err := c.GetApp(opts)
 	if err != nil {
 		return err
 	}
-	json, err := json.MarshalIndent(manifest.Attrs, "", "  ")
+	newManifest, err := c.UpdateApp(opts, flagSafeUpdate)
 	if err != nil {
 		return err
 	}
-	fmt.Println(string(json))
+	msg := "%s is already up-to-date at %s"
```

Comment:
Why this message has no `\n`, but the message for an updating app has one?

Reply:
It's a mistake. I forgot it.